src/sentry/models/apiauthorization.py

```diff
@@ -35,7 +35,7 @@ class ApiAuthorization(Model, HasApiScopes):
     class Meta:
         app_label = "sentry"
         db_table = "sentry_apiauthorization"
-        unique_together = (("user", "application"),)
+        unique_together = (("user", "application", "organization_id"),)
```

Comment:
You might want to do this in multiple steps. First add the new unique index, and then remove the `user, application` index.

Reply:
@markstory can you provide more context on why multiple steps?

Reply:
It's because the migration runs it like: 
```
ALTER TABLE "sentry_apiauthorization" DROP CONSTRAINT "sentry_apiauthorization_user_id_application_id_f7975a32_uniq";
CREATE UNIQUE INDEX CONCURRENTLY "sentry_apiauthorization_user_id_application_id_o_da61c118_uniq" ON "sentry_apiauthorization" ("user_id", "application_id", "organization_id");
ALTER TABLE "sentry_apiauthorization" ADD CONSTRAINT "sentry_apiauthorization_user_id_application_id_o_da61c118_uniq" UNIQUE USING INDEX "sentry_apiauthorization_user_id_application_id_o_da61c118_uniq";
```
So there's a period of time where the unique constraint isn't enforced. So this migration might be better written in a way that it adds the new unique first, then drops the old one